recipes/r-proteus-bartongroup/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "r-proteus-bartongroup" %}
+{% set version = "0.2.16" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  url: https://github.com/bartongroup/Proteus/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: befd66aeb5c29ea10d1c457aad9e47f3f757347e8c60734a8f01161222718137
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-dplyr
+    - r-reshape2
+    - r-ggplot2
+    - r-ggdendro
+    - r-ggrepel
+    - r-ggExtra
+    - r-gridExtra
+    - r-viridis
+    - r-hexbin
+    - bioconductor-limma
+    - r-shiny
+    - r-dt
+  run:
+    - r-base
+    - r-dplyr
+    - r-reshape2
+    - r-ggplot2
+    - r-ggdendro
+    - r-ggrepel
+    - r-ggExtra
+    - r-gridExtra
+    - r-viridis
+    - r-hexbin
+    - bioconductor-limma
+    - r-shiny
+    - r-dt
+
+test:
+  commands:
+    - $R -e "library('proteus'); proteus::proteinColumns"           # [not win]
+    - "\"%R%\" -e \"library('proteus'); proteus::proteinColumns\""  # [win]
```

Comment:
This line can again be removed, the `not win` comment above can also be removed.